### PR TITLE
Add ImageDigestMirrorSet resource to interact with idms resource in a…

### PR DESCRIPTION
**Pull request by dbasunag**

### CODE REVIEW

File: ocp_resources/image_digest_mirror_set.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+from ocp_resources.resource import Resource
+
+
+class ImageDigestMirrorSet(Resource):
+    """
+    Create/Manage ImageDigestMirrorSet configuration object. API reference:
+    https://docs.openshift.com/container-platform/4.14/rest_api/config_apis/imagedigestmirrorset-config-openshift-io-v1.html
+    Args:
```
Comment (myakove) [inline ocp_resources/image_digest_mirror_set.py:8]: In class dockstring: `API reference: https://docs.openshift.com/container-platform/4.14/rest_api/config_apis/imagedigestmirrorset-config-openshift-io-v1.html`

in __init__: 
```
Create/Manage ImageDigestMirrorSet configuration object

Args: .....
```

:)

File: ocp_resources/image_digest_mirror_set.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        imageDigestMirrors (list of dict):
+            e.g. [{source: <str>, mirrors: <list>}, ..., {source: <str>, mirrors: <list>}]
+            - source - the repository that users refer to, e.g. in image pull specifications
+            - mirrors - one or more repositories (str) that may also contain the same images. The order
+                of mirrors in this list is treated as the user’s desired priority
+    """
+
+    api_version = "v1"
```
Comment (myakove) [inline ocp_resources/image_digest_mirror_set.py:16]: We should get `api_version` automatically from the group

**Review by myakove — CHANGES_REQUESTED** [review]

File: ocp_resources/image_digest_mirror_set.py (hunk 1 above)
Comment (dbasunag) [reply ocp_resources/image_digest_mirror_set.py:8]: Sorry! Misunderstood. Will change.

File: ocp_resources/image_digest_mirror_set.py (hunk 2 above)
Comment (dbasunag) [reply ocp_resources/image_digest_mirror_set.py:16]: Ok, I see what you mean.

**Review by omrirh — APPROVED** [review]

File: ocp_resources/image_digest_mirror_set.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    """
+    API reference:
+    https://docs.openshift.com/container-platform/4.14/rest_api/config_apis/imagedigestmirrorset-config-openshift-io-v1.html
+    """
+
+    api_group = Resource.ApiGroup.CONFIG_OPENSHIFT_IO
+
+    def __init__(self, image_digest_mirrors=None, **kwargs):
+        """
+        Create/Manage ImageDigestMirrorSet configuration object. API reference:
```
Comment (rnetser) [inline ocp_resources/image_digest_mirror_set.py:14]: api ref should be removed

File: ocp_resources/image_digest_mirror_set.py [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+        https://docs.openshift.com/container-platform/4.14/rest_api/config_apis/imagedigestmirrorset-config-openshift-io-v1.html
+        Args:
```
Comment (rnetser) [inline ocp_resources/image_digest_mirror_set.py:16]: missing new line before `Args`

**Review by rnetser — CHANGES_REQUESTED** [review]

File: ocp_resources/image_digest_mirror_set.py (hunk 3 above)
Comment (dbasunag) [reply ocp_resources/image_digest_mirror_set.py]: Sorry about that.

File: ocp_resources/image_digest_mirror_set.py [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+    """
+
+    api_group = Resource.ApiGroup.CONFIG_OPENSHIFT_IO
+
+    def __init__(self, image_digest_mirrors=None, **kwargs):
+        """
+        Create/Manage ImageDigestMirrorSet configuration object.
+
+        Args:
+            imageDigestMirrors (list of dict):
```
Comment (rnetser) [inline ocp_resources/image_digest_mirror_set.py:17]:
```suggestion
            image_digest_mirrors (list of dict):
```
sorry, missed this before

**Review by rnetser — CHANGES_REQUESTED** [review]

**Review by rnetser — APPROVED** [review]

**Review by myakove — APPROVED** [review]

**Pull request merged**